source/file/Exporter.js

```diff
@@ -60,6 +61,43 @@ class Exporter {
     }
   }
 
+  toSvg(tiles) {
+    // create svg
+    const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
+    const canv = document.getElementById('canvas').getElementsByTagName('canvas')[0]
+    const width = canv.width
+    const height = canv.height
+    svg.setAttribute('width', width)
```

Comment:
Just curious, is there a reason to use `setAttribute` vs just setting the attribute? e.g. `svg.width = width`, we use setAttribute all over this project and I just don't get why? I thought they were mostly equivalent?